posts/2020-05-07-Rust.1.43.1.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "Announcing Rust 1.43.1"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team has published a new point release of Rust, 1.43.1.
+Rust is a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup, getting Rust 1.43.1 is as easy as:
+
+```console
+rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.43.1][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1431-2020-05-07
+
+
+## What's in 1.43.1
+
+Rust 1.43.1 addresses two regressions in 1.43.0, and updates the OpenSSL version
+shipped with Cargo.
+
+### OpenSSL version bump to 1.1.1g
+
+This fixes CVE-2020-1967 in OpenSSL.
+
+TODO: Say more? Unclear what the impact etc was.
```

Comment:
According to https://nvd.nist.gov/vuln/detail/CVE-2020-1967 the other end of the connection can cause a null pointer dereference, thus causing a crash.